src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentAction.java

```diff
@@ -67,18 +67,35 @@ protected void loadDocument(KualiDocumentFormBase kualiDocumentFormBase) {
         IWantDocumentForm iWantForm = (IWantDocumentForm) kualiDocumentFormBase;
         IWantDocument iWantDocument = iWantForm.getIWantDocument();
 
-        if (iWantDocument.getDocumentHeader().getWorkflowDocument().isSaved()) {
-            iWantForm.setStep(CUPurapConstants.IWantDocumentSteps.CUSTOMER_DATA_STEP);
-            iWantDocument.setStep(CUPurapConstants.IWantDocumentSteps.CUSTOMER_DATA_STEP);
+        if (StringUtils.equalsIgnoreCase(iWantForm.getPresentationMode(),
+                CUPurapConstants.IWantPresentationModes.MULTIPLE_PAGE_MODE)) {
             
-            if (StringUtils.isNotBlank(iWantDocument.getCurrentRouteToNetId())) {
-                iWantForm.getNewAdHocRoutePerson().setId(iWantDocument.getCurrentRouteToNetId());
+            if (iWantDocument.getDocumentHeader().getWorkflowDocument().isSaved()) {
+
+                iWantForm.setStep(CUPurapConstants.IWantDocumentSteps.CUSTOMER_DATA_STEP);
+                iWantDocument.setStep(CUPurapConstants.IWantDocumentSteps.CUSTOMER_DATA_STEP);
+
+                if (StringUtils.isNotBlank(iWantDocument.getCurrentRouteToNetId())) {
+                    iWantForm.getNewAdHocRoutePerson().setId(iWantDocument.getCurrentRouteToNetId());
+                }
+
+            } else if (!iWantDocument.getDocumentHeader().getWorkflowDocument().isInitiated()) {
+              iWantForm.setStep(CUPurapConstants.IWantDocumentSteps.REGULAR);
+              iWantDocument.setStep(CUPurapConstants.IWantDocumentSteps.REGULAR);
```

Comment:
There are a couple lines here where the indentation is still off a bit.

Reply:
fixed.

Reply:
Formatting has been fixed and unit tests are running.